tests/robustness/main_test.go

```diff
@@ -149,7 +149,10 @@ func runScenario(ctx context.Context, t *testing.T, s scenarios.TestScenario, lg
 		return nil
 	})
 	g.Wait()
-	return append(operationReport, append(failpointClientReport, watchReport...)...)
+
+	// we query for revision 0, which asks the hash to be computed on all keys
+	hashKvReport := client.CheckHashKV(ctx, t, clus, 0, baseTime, ids)
```

Comment:
This function represents high level structure of the test, don't think checking hash makes sense here. Could you move it this to traffic.SimulateTraffic?